src/locales/en/common.json

```diff
@@ -2189,6 +2243,69 @@
       }
     }
   },
+  "signTransaction": {
```

Comment:
is there a way we can avoid completely duplicating the entry here and reuse the one from send as much as possible?

Reply:
It's what was done for desktop I believe